### ISSUE TITLE
Phone number lookup fails with 'fraud' KeyError

### ISSUE THREAD

**Issue by jeremymailen** [twilio/twilio-python#410]
**Version:**
6.10.4

### Code Snippet
```python
from twilio.rest import Client

twilio = Client(account_sid, auth_token)
lookup = twilio.lookups.phone_numbers(number).fetch()
```

### Exception/Log
```
    lookup = twilio.lookups.phone_numbers(number).fetch()
  File "/usr/local/lib/python3.6/site-packages/twilio/rest/lookups/v1/phone_number.py", line 150, in fetch
    return PhoneNumberInstance(self._version, payload, phone_number=self._solution['phone_number'], )
  File "/usr/local/lib/python3.6/site-packages/twilio/rest/lookups/v1/phone_number.py", line 187, in __init__
    'fraud': payload['fraud'],
KeyError: 'fraud'```

### Steps to Reproduce
1. See above

**Comment by rossgoodwin** [comment]
I am experiencing this error as well. Did you find a solution?

**Comment by jeremymailen** [comment]
I switched back to `6.10.3` which doesn't have this bug.

**Comment by masterteedy** [comment]
I have the problem in the version released today too (10.6.5).

**Comment by alfozan** [comment]
I upgraded to version 6.10.5 and the issue is still there. Current solution to comment out this line:
https://github.com/twilio/twilio-python/blame/master/twilio/rest/lookups/v1/phone_number.py#L187

**Comment by alfozan** [comment]
looks like the issue was fixed in the last release : 6.11.0
